app/(header-default)/entreprise/[slug]/page.tsx

```diff
@@ -67,7 +67,7 @@ export const generateMetadata = withErrorHandler(
 export default withErrorHandler(async function UniteLegalePage(
   props: AppRouterProps
 ) {
-  const session = await useSessionServer();
+  const session = await getSession();
```

Comment:
I renamed this, because its not a hook technically, and thus, not concerned by the [rules of hook](https://react.dev/warnings/invalid-hook-call-warning)